fixity/utils.py

```diff
@@ -18,7 +19,7 @@ def check_valid_uuid(uuid):
     This uses the UUID() class constructor from the uuid module,
     which raises a ValueError if the passed string is not a valid UUID.
     """
-    if not isinstance(uuid, basestring):
+    if not isinstance(uuid, six.string_types):
```

Comment:
Updated to generate string_types internally, then discovered six is a dependency of vcrpy, so it'll be installed in most cases anyway.